### PR TITLE
FE-2 add default ESLint ruleset to laws-of-the-game

### PR DESCRIPTION
**Pull request by alxndr**
https://statmilk.atlassian.net/browse/FE-2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,14 @@
     "test": "mocha tests/ --recursive"
   },
   "dependencies": {
+    "eslint-plugin-babel": "4.1.2",
```
Comment (pcanterini) [inline package.json]: Do we want only some of these versions pinned but not others?

Comment (alxndr) [reply package.json]: Not really sure. I suppose we can pin them all, yeah?

Comment (pcanterini) [reply package.json]: Sounds like a plan :)

**Comment by alxndr** [comment]
How should we handle version bumps and, in general, deploying this?

**Comment by alxndr** [comment]
This will be a Minor level version bump, i.e. `1.2.0` -> `1.3.0`

**Pull request merged**